Psl criteria checklist

elkeasen opened the pull request.

elkeasen committed:
Adding License

Added a markdown file that has the required license language in it (Creative Commons Public Domain Dedication (CC0)).

elkeasen committed:
Added Contact Information and Suggested Citation

Added sections "Contact Information" and "Suggested Citation" at the end of the README.

elkeasen committed:
Added "Latest Release"

Added a section "Latest Release" in the README (using semantic versioning scheme).

elkeasen committed:
Add Installation Directions

Added a section "Installation Directions" in the README that briefly explain how to clone a GitHub repository and how to install R.

elkeasen committed:
Minor Change to Installation Directions

Added a sentence to the installation directions.

tvanantwerp commented on README.md:
```diff
@@ -2,6 +2,24 @@
 The Tax Foundation’s publication [Capital Cost Recovery across the OECD](https://taxfoundation.org/publications/capital-cost-recovery-across-the-oecd/) shows how capital allowances compare across OECD countries and how they have developed since 2000. The dataset we compiled for the years 1979 to 2020 is made available as a resource for research.
 
+## Latest Release
+The current version of this project is 1.1.0 (2021-03-31).
```
GitHub actually has a feature for creating and tracking releases—try it [here](https://github.com/TaxFoundation/capital-cost-recovery/releases/new)

elkeasen replied:
That's awesome -- just added it. Thanks!

tvanantwerp commented on README.md:
```diff
@@ -2,6 +2,24 @@
+### Cloning the GitHub repository
+To install capital-cost-recovery, run the following in the folder you wish to store the files:
+
+git clone https://github.com/TaxFoundation/capital-cost-recovery
+cd capital-cost-recovery
```
```recommendation
git clone https://github.com/TaxFoundation/capital-cost-recovery
cd capital-cost-recovery
```

elkeasen replied:
Not sure I understand what exactly you mean here.

tvanantwerp replied:
It's a recommendation to put the text in code blocks (three backticks above and below) so that it looks like this...

<img width="481" alt="image" src="https://user-images.githubusercontent.com/2487968/125120490-7a972300-e0c0-11eb-90d8-bef6a0f8191d.png">

...rather than like this...

<img width="666" alt="image" src="https://user-images.githubusercontent.com/2487968/125120526-88e53f00-e0c0-11eb-827a-505e66258f2a.png">

The recommendation itself is in backticks, so GitHub might not be interpreting my meaning.

elkeasen replied:
That makes sense, thank you!

tvanantwerp commented on README.md:
```diff
@@ -2,6 +2,24 @@
+git clone https://github.com/TaxFoundation/capital-cost-recovery
+cd capital-cost-recovery
+
+More details on cloning a repository can be found here: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/cloning-a-repository.
```
Could use a markdown link like so:

```
More details on cloning a repository can be found [here](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/cloning-a-repository).
```

tvanantwerp commented on README.md:
```diff
@@ -2,6 +2,24 @@
+More details on cloning a repository can be found here: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/cloning-a-repository.
+
+### Installing R
+If you haven't already done so, download RStudio here: https://www.rstudio.com/products/rstudio/.
```
Another opportunity for a markdown link:

```
If you haven't already done so, download and install [RStudio](https://www.rstudio.com/products/rstudio/).
```

tvanantwerp commented on README.md:
```diff
@@ -80,4 +98,12 @@ Location of **output tables** that are included in the publication or needed to
+
+
+## Contact Information
+This repository is maintained by Daniel Bunn at Tax Foundation (dbunn@taxfoundation.org).
```
```recommendation
This repository is maintained by [Daniel Bunn](mailto:dbunn@taxfoundation.org) at the [Tax Foundation](https://taxfoundation.org).
```

tvanantwerp commented on README.md:
```diff
@@ -2,6 +2,24 @@
+
+git clone https://github.com/TaxFoundation/capital-cost-recovery
+cd capital-cost-recovery
+
```
I would also add whatever command is necessary to run the program from the command line, assuming somebody prefers that to using RStudio.

elkeasen replied:
There is a way to run the R code without using RStudio?

tvanantwerp replied:
It looks like `Rscript <<name of the file>>` would do it, but I'm not positive. Googling gave a variety of strange answers. R is weird.

elkeasen replied:
Haha ok, I'll stick with not including it then. Just to make sure I'm not including something that may not work for everyone.

elkeasen committed:
Added PSL Catalog and minor README updates

Added .json file for the PSL catalog (https://pslmodels.org/Catalog/index.html) and made some improvements to the README file.
